tests/Dfe.EarlyYearsQualification.UnitTests/Extensions/MockLoggerExtensions.cs

```diff
@@ -0,0 +1,19 @@
+using Microsoft.Extensions.Logging;
+using Moq;
+
+namespace Dfe.EarlyYearsQualification.UnitTests.Helpers;
+
+public static class MockLoggerExtensions
+{
+  public static void VerifyError<T>(this Mock<ILogger<T>> _logger, string message)
```

Comment:
Can we rename this to `logger` for consistency of method param names, and the inner one to `logger1`?

Reply:
…actually, an even better idea, rename the method parameter to `mockLogger` (to make its intention more obvious), then you don't even have to rename the inner variable 🙂